packages/x-components/src/components/__tests__/dynamic-props.mixin.spec.ts

```diff
@@ -0,0 +1,41 @@
+import Vue from 'vue';
+import { mount, Wrapper } from '@vue/test-utils';
+import { ExtractArrayItems } from '@empathyco/x-utils';
+import { dynamicPropsMixin } from '../dynamic-props.mixin';
+
+const renderComponent = ({
+  props = ['list', 'button'] as const
+}: ComponentOptions = {}): ComponentAPI => {
+  const wrapper = mount({
+    mixins: [dynamicPropsMixin<ExtractArrayItems<typeof props>>(props)],
+    render: h => h(),
+    props: ['data']
+  });
+  return { wrapper };
+};
+
+describe('dynamicPropsMixin', () => {
+  it('expects to have the defined props from the mixin', () => {
+    const { wrapper } = renderComponent();
+    const props = Object.keys(wrapper.props());
+    expect(props).toHaveLength(3);
+    expect(props.includes('list')).toBeTruthy();
+    expect(props.includes('button')).toBeTruthy();
+    expect(props.includes('data')).toBeTruthy();
```

Comment:
Does it make sense to "simplify" this assertions like this?
```suggestion
    expect(props).toEqual(['list', 'button', 'data']);
```

Reply:
There is a reason to do it the other way. With `.toEqual` you are also implicitly testing the order, which some times like this one we do not control. I don't think this will change in the short term, so we can leave it like you suggested.